test/library/packages/LinearAlgebra/performance/jacobi-perf.perfkeys

```diff
@@ -0,0 +1,3 @@
+LinearAlgebra.jacobi:
+LinearAlgebra.Sparse.jacobi:
+
```

Comment:
this extra line breaks `start_test`. You can test your perf tests by running `start_test --performance` on them